docs/modules/getting-started/pages/install-hazelcast.adoc

```diff
@@ -34,40 +34,53 @@ Linux::
 +
 --
 
-To install on Linux, you can use either of these package managers, depending on your Linux distribution:
+. Check that you have the Wget utility installed. Hazelcast uses this utility in its package managers.
++
+```bash
+wget -v
+```
++
+If you do not see a version number, download and install link:https://www.gnu.org/software/wget/[Wget^].
++
+
+. Use either of these package managers to install Hazelcast, depending on your Linux distribution:
 ifdef::snapshot[]
++
```

Comment:
I think we can remove this + and put it before the first conditional.

Reply:
Updated as requested.